cudl-sandbox/terraform.tfvars

```diff
@@ -140,14 +178,17 @@ transform-lambda-information = [
     "mount_fs"                 = false
     "environment_variables" = {
       ANT_TARGET            = "full"
-      COLLECTION_XML_SOURCE = "/tmp/opt/cdcp/dist-pending/collection-xml"
-      CORE_XML_SOURCE       = "/tmp/opt/cdcp/dist-pending/core-xml"
-      PAGE_XML_SOURCE       = "/tmp/opt/cdcp/dist-pending/page-xml"
+      #COLLECTION_XML_SOURCE = "/tmp/opt/cdcp/dist-pending/collection-xml"
+      #CORE_XML_SOURCE       = "/tmp/opt/cdcp/dist-pending/core-xml"
+      #PAGE_XML_SOURCE       = "/tmp/opt/cdcp/dist-pending/page-xml"
```

Comment:
These lines could be removed as they will still exist in git history

Reply:
Removed.